src/pages/Routes.tsx

```diff
@@ -24,6 +24,7 @@ export function Routes() {
   return (
     <Switch>
       <Route path="swap" element={<Swap />} />
+      <Route path="swap/pro" element={<Swap />} />
```

Comment:
if we have a different url for pro trading, why add "pro" to state?

Reply:
It was the first solution when the page was the same. Question if we stay with the new URL or not. Also, we can remember which Tab was selected and redirect on pressing tab in the nav but it is for discussion

Reply:
if we stay with 'swap/pro' and no redirect so we will remove it from the state